README.md

```diff
@@ -53,12 +53,18 @@ example of the `lcov` format [here](./lcov.md).
 cairo-coverage path/to/trace/1.json path/to/trace/2.json path/to/trace/3.json
 ```
 
-### Coverage statistics
+### Usage locally
 
-Various tools exist that can produce coverage statistics based on an lcov file. One of the most well-known among these
-is [genhtml](https://github.com/linux-test-project/lcov/blob/master/bin/genhtml),
-a tool that generates an html summary report using the coverage data in an lcov file, `genhtml` is part of
-the [lcov package](https://github.com/linux-test-project/lcov/tree/master).
+A summary report with aggregated data can be produced by one of many tools that accept the `lcov` format.
+In this example we will use the `genhtml` tool from the [lcov package](https://github.com/linux-test-project/lcov/tree/master) to generate an HTML report.
```

Comment:
nit: repo does not state that, but it's actually available with most of (if not all) package managers like brew, apt or pacman. Maybe it is a good idea to add just a vague information that it is the case? I can see people being frightened by the `make` command or whatever

Reply:
Maybe change link to this `https://command-not-found.com/genhtml` would fix it?

Reply:
![obraz](https://github.com/user-attachments/assets/2f1437cc-de97-44f3-8504-98546d551ddc)

this is thew first time I see this page it's cool!

Reply:
But I think that also giving an url to git next to this is a good idea :+1:

Reply:
Added thanks for suggestion 😎